<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageHistory</class>
 <widget class="QWidget" name="PageHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dex</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;
color:black;</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FCD535;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #474D57;</string>
    </property>
    <property name="text">
     <string>History</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235,235,235);
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../../res.qrc">
      <normaloff>:/res/ic/res/ic/arowLeft.png</normaloff>:/res/ic/res/ic/arowLeft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>289</width>
      <height>149</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="previousPagePushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235,235,235);
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../res.qrc">
     <normaloff>:/res/ic/res/ic/arrowLeft1.png</normaloff>:/res/ic/res/ic/arrowLeft1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPagePushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235,235,235);
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../res.qrc">
     <normaloff>:/res/ic/res/ic/arrowRight1.png</normaloff>:/res/ic/res/ic/arrowRight1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ofPagesLabel">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #474D57;</string>
   </property>
   <property name="text">
    <string>100 of 100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
